### ISSUE TITLE
Got different accuracy results

### ISSUE THREAD

**Issue by Sungwon-Han** [JordanAsh/badge#1]
Thank you for sharing the source code for this great work.

I am trying to replicate some of your baseline results. 
However, the result from the baseline (LeastConfidence) I got is much better than the one reported in your paper.
Here is my result accuracy plot (It achieves 40% test accuracy much earlier than the reported accuracy in the paper).

Query size: 100
Dataset: CIFAR-10
Network: Resnet
Strategy: LeastConfidence
![image](https://user-images.githubusercontent.com/17738624/88863172-25f6f100-d23d-11ea-9933-c0848eb44727.png)

Should I do something to replicate your results?

**Comment by JordanAsh** [comment]
Yes, the default learning rate isn't correct. It should be an order of
magnitude lower, I believe.

**Comment by Sungwon-Han** [comment]
Even after changing the learning rate as same as reported in the paper, I can't replicate the paper's result.
Are there any other things that I should do for replication?

**Comment by JordanAsh** [comment]
Right, I'm saying that I believe the learning rate in the paper is off by
an order of magnitude.

**Comment by Sungwon-Han** [comment]
Then, can you provide a command to replicate the paper's result?
Or, are there any seeds to replicate the paper's result?
